src/features/projects/components/ProjectTypeIcon.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import NaturalRegenerationIcon from '../../../../public/assets/images/icons/project/NaturalRegenerationIcon';
+interface Props {
+  projectType: string;
+}
+const ProjectTypeIcon = ({ projectType }: Props) => {
+  switch (projectType) {
+    case 'natural-regeneration':
+      return <NaturalRegenerationIcon />;
```

Comment:
Do we have any example project on staging to see how this looks like?

Reply:
Now I see some :-)
<img width="569" alt="Bildschirmfoto 2023-06-10 um 12 09 30" src="https://github.com/Plant-for-the-Planet-org/planet-webapp/assets/1532418/9b050a71-7d3b-4e66-bc2d-06fe46febf5e">